admin/release.py

```diff
@@ -67,3 +92,212 @@ def make_rpm_version(flocker_version):
         release.extend(remainder)
 
     return rpm_version(version, '.'.join(release))
+
+
+class NotTagged(Exception):
+    """
+    Raised if publishing to production and the published version isn't tagged.
```

Comment:
Clarify that the untagged version is not the version which has been published, but instead the version which the code is trying to publish.